alicloud/table_alicloud_rds_instance_metric_cpu_utilizations.go

```diff
@@ -0,0 +1,38 @@
+package alicloud
```

Comment:
@ParthaI can you please rename the file to 
`table_alicloud_rds_instance_metric_cpu_utilization`?